### ISSUE TITLE
schema type for auditlog.responseCode doesnt match actual returned value type

### ISSUE THREAD

**Issue by cjellick** [rancher/rancher#9890]
**Rancher versions:**
rancher/server: 1.6.9
rancher/agent:

Here's the schema for auditLog:
```
{
"id": "auditLog",
"type": "schema",
...
"resourceFields": {
...
"responseCode": {
"type": "string",
"description": null,
"create": false,
"update": false
...
```
but here's an actual returned audit log objet:
```
{
"id": "1al13",
"type": "auditLog",
...
"responseCode": 201,
...
}
```

We store it in the DB as an int here: https://github.com/rancher/cattle/blob/5a0f58013ba8ea185fe7f1246a4ab44646c16752/modules/caas/common/src/main/java/io/cattle/platform/audit/impl/AuditServiceImpl.java#L76
which means the data is _stored_ incorrectly. So, we either need to change the schema type from string to int (and regenerate the golang client (which is how we found this bug) or we need to create an output filter that dynamically changes the value from an int to a string to match the existing schema

**Comment by vincent99** [comment]
But it **is** an int... so isn't it stored & returned correctly and the schema type is just wrong?

**Comment by cjellick** [comment]
@vincent99 sorry, bad wording on my part. The data is stored *inconsistently with its representation in the schema*.

I'm open for changing the schema type, which is the quickest and easiest fix, but wanted to get at least you or @ibuildthecloud to agree since we are technically breaking the schema

**Comment by cjellick** [comment]
Offline discussion: we are in agreement that we should just change the schema to be an string.

**Comment by cjellick** [comment]
available on master and v1.6-development. not in an RC yet

**Comment by galal-hussein** [comment]
Rancher server: v1.6.11-rc1

I can verify that the responseCode type is now int instead of string in the schema:
```
"responseCode": {
"type": "int",
"description": null,
"create": false,
"update": false,
"readOnCreateOnly": false
},
```

and no issues found when displaying audit logs.
